ENGAGEHF/Dashboard/Dashboard.swift

```diff
@@ -6,36 +6,31 @@
 // SPDX-License-Identifier: MIT
 //
 
-import SpeziMockWebService
 import SwiftUI
 
 
-struct MockUpload: View {
+struct Dashboard: View {
     @Binding var presentingAccount: Bool
     
+    
     var body: some View {
         NavigationStack {
-            RequestList()
+            VStack {
+                Greeting()
+                Spacer()
+            }
+                .navigationTitle("Home")
                 .toolbar {
-                    if AccountButton.shouldDisplay {
+                    ToolbarItem(placement: .topBarTrailing) {
```

Comment:
I would suggest to stick with the `if AccountButton.shouldDisplay` logic here as this is required to hide the button if the firebase connection is disabled (e.g. using the feature flag) when the Account button should not be shown.
```suggestion
                    if AccountButton.shouldDisplay {
```